test/e2e/scenarios/start/benefitType.js

```diff
@@ -27,31 +28,22 @@ Scenario('When I enter a non PIP benefit type, I am taken to the download form p
 
 });
 
-const benefit = new DataTable(['BenefitType', 'BenefitName']); //
-benefit.add(['Attendance Allowance', 'Attendance Allowance']); // adding records to a table
-benefit.add(['Bereavement Benefit', 'Bereavement Benefit']);
-benefit.add(['Carer’s Allowance', 'Carer’s Allowance']);
-benefit.add(['Disability Living Allowance (DLA)', 'Disability Living Allowance (DLA)']);
-benefit.add(['Employment and Support Allowance (ESA)', 'Employment and Support Allowance (ESA)']);
-benefit.add(['Incapacity Benefit', 'Incapacity Benefit']);
-benefit.add(['Industrial Injuries Disablement', 'Industrial Injuries Disablement']);
-benefit.add(['Jobseeker’s Allowance (JSA)', 'Jobseeker’s Allowance (JSA)']);
-benefit.add(['Maternity Allowance', 'Maternity Allowance']);
-benefit.add(['Severe Disablement Allowance', 'Severe Disablement Allowance']);
-benefit.add(['Social Fund', 'Social Fund']);
-benefit.add(['Universal Credit (UC)', 'Universal Credit (UC)']);
-
-Data(benefit).Scenario('When I enter differernt non PIP benefit types, I am taken to the download form page', (I, current) => {
-    I.enterBenefitTypeAndContinue(current.BenefitName);
-    I.see(current.BenefitName, {css: '.column-two-thirds>p'});
-    I.seeInCurrentUrl(paths.identity.downloadAppointeeForm);
-    I.see('Continue to form download');
-    I.click({css: '.link-back'});
+Scenario('When I enter differernt non PIP benefit types, I am taken to the download form page', (I) => {
+    let i;
+    for(i=0; i<types.length; i++){
```

Comment:
This is OK. However, the solution provided by Paul looks to me more elegant, tbh.

Reply:
Yes, as per the Paul i have changed it, i will leave it you both to approve this.

Reply:
This implementation hasn't followed our discussion - I've even provided the code to help. Please reimplement this as per the discussion.